--- a/Makefile.am       2013-04-06 14:46:37.955969119 +0200
+++ b/Makefile.am       2013-04-06 14:46:51.394965657 +0200
@@ -38,7 +38,7 @@
 if NEED_LIBBLKMAKER
 SUBDIRS           += libblkmaker
 bfgminer_CPPFLAGS += -Ilibblkmaker
-bfgminer_LDFLAGS  += -Llibblkmaker/.libs -Wl,-rpath,libblkmaker/.libs
+bfgminer_LDFLAGS  += -Llibblkmaker/.libs
 bfgminer_LDADD    += -lblkmaker_jansson-0.1 -lblkmaker-0.1

 if HAVE_CYGWIN
