--- a/Makefile.in	2013-04-04 18:03:11.198185097 +0200
+++ b/Makefile.in	2013-04-04 18:03:39.020202287 +0200
@@ -57,7 +57,7 @@
 bin_PROGRAMS = bfgminer$(EXEEXT) $(am__EXEEXT_1) bfgminer-rpc$(EXEEXT)
 @NEED_LIBBLKMAKER_TRUE@am__append_1 = libblkmaker
 @NEED_LIBBLKMAKER_TRUE@am__append_2 = -Ilibblkmaker
-@NEED_LIBBLKMAKER_TRUE@am__append_3 = -Llibblkmaker/.libs -Wl,-rpath,libblkmaker/.libs
+@NEED_LIBBLKMAKER_TRUE@am__append_3 = -Llibblkmaker/.libs
 @NEED_LIBBLKMAKER_TRUE@am__append_4 = -lblkmaker-0.1 -lblkmaker_jansson-0.1
 @HAVE_CYGWIN_TRUE@@NEED_LIBBLKMAKER_TRUE@am__append_5 = cygblkmaker-0.1-0.dll cygblkmaker_jansson-0.1-0.dll
 @HAS_SCRYPT_TRUE@am__append_6 = scrypt.c scrypt.h
